content/guides/installing/making_haiku_usb_stick.html

```diff
@@ -61,7 +61,9 @@ <h1><a id="#haiku">Haiku: Using Installer and DriveSetup</a></h1>
 
 <p>The last step is to go to <span class="menu">Partition | Format | Be File System...</span>, choose a name and format the new partition.</p>
 
-<p>Close DriveSetup and launch Installer.</p>
+<p>Close DriveSetup and launch Installer (found under <span class="menu">Applications</span> in the Deskbar).</p>
+
+<p><img src="/files/guides/installing/usb_installer_onto.png" alt="Install Haiku Onto the new partition" /></p>
```

Comment:
I'm also not sure what value this image is...

Reply:
All other steps are documented with pictures besides this final (and important) step. If a user is trying to install quickly or switching back and forth between screens often, I hypothesize they potentially could miss this final step and believe the instructions did not work.

Reply:
I agree, @Danc2050.